students/stachowska_agata/lesson_06_dicts_tuples_sets_args_kwargs/args_inspector.py

```diff
@@ -0,0 +1,5 @@
+def inspect_args(*args, **kwargs):
+    for arg in args:
+        print(arg)
+    for key, value in kwargs.items():
+        print(key, value, sep=': ')
```

Comment:
Missing direct call of inspect_args() function

e.g.
inspect_args(1, [2, 3], keyword_arg="test_arg")